CMakeLists.txt

```diff
@@ -347,7 +347,8 @@ find_package(SWIG)
 
 if(SWIG_FOUND)
     set(SWIG_VERSION_CHECK FALSE)
-    if("${SWIG_VERSION}" VERSION_GREATER ${GR_SWIG_MIN_VERSION})
+    if("${SWIG_VERSION}" VERSION_EQUAL ${GR_SWIG_MIN_VERSION} OR
+       "${SWIG_VERSION}" VERSION_GREATER ${GR_SWIG_MIN_VERSION})
```

Comment:
Let me look, at least on next it was set to 2.0.11, which was causing the fail.  I didn't check on master.